elasticai_api/common/data_shard_service.py

```diff
@@ -45,12 +44,14 @@ def __init__(
         batch_size,
         num_epochs=None,
         dataset_size=None,
+        shuffle=False,
         task_type=elasticai_api_pb2.TRAINING,
     ):
         self._mc = master_client
         self._batch_size = batch_size
         self._num_epochs = num_epochs
         self._dataset_size = dataset_size
+        self._shuffle = shuffle
```

Comment:
I think we can name it element_wise_shuffle or per_record_shuffle.

Reply:
`shuffle` is a common word in PyTorch and TensorFlow. Here, I want to keep the same as these frameworks and not introduce a new word to users. I think we should use the argument `shuffle_shards` to shuffle the shard list for training.

Reply:
‘shuffle_shards’ is a good idea.

Reply:
Yeah, and we can add the argument `shuffle_shards` in the next PR.